adafruit_azureiot/device_registration.py

```diff
@@ -49,7 +49,7 @@ class DeviceRegistrationError(Exception):
     """
 
     def __init__(self, message):
-        super(DeviceRegistrationError, self).__init__(message)
```

Comment:
Why are you removing `DeviceRegistrationError` from the superclass?

Reply:
Same as above